public/app/features/migrate-to-cloud/onprem/Page.tsx

```diff
@@ -232,3 +247,13 @@ export const Page = () => {
     </>
   );
 };
+
+function maybeGetTraceID(err: unknown): string | undefined {
```

Comment:
```suggestion
function maybeGetTraceID(err: unknown) {
```
Don't think the return type is necessary, it is correctly inferred.